### ISSUE TITLE
5841081A - Super Contra

### ISSUE THREAD

**Issue by SonofUgly** [xenia-project/game-compatibility#329]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Super-Contra/66acd000-77fe-1000-9115-d8025841081a)
Tested on benvanik/xenia@826ea60

Repeating/overlapping graphics in-game
# Log:
[Log](http://www.mediafire.com/download/nh102hjjr6kpzxv/xenia_-_Super_Contra.log) (16MB)
# Screenshot:
![](http://i.imgur.com/SJx4edH.png)

**Comment by Razzile** [comment]
this is a pretty unique graphical error ka ka. not seen this before 🍩

**Comment by SonofUgly** [comment]
Yeah, it's definitely a strange one, even the pause menu gets repeated like that. [Cilp](https://u.pomf.io/yqccvg.webm) just cause.
